GoogleForCreators/web-stories-wp
# Borders: Remove inner and center border

## Summary
Removes inner and center border, keeping only the outside border as the default.


## Relevant Technical Choices



## To-do



## User-facing changes
- Inner and center border have been removed, the border is displayed only as outside-border.
- Moved the logic from making page carousel preview responsive to the outside border as well.


## Testing Instructions
- Add an element, e.g. media.
- Add some border, verify there is no option for choosing border type.
- Verify the border is displayed outside of the selection line
- Verify it works as expected with radius, color, etc.


---



Fixes #5529
